include/irr/asset/ICPUBuffer.h

```diff
@@ -18,6 +18,14 @@ namespace irr
 namespace asset
 {
 
+//! One of CPU class-object representing an Asset
+/**
+	An Asset is a CPU object which needs to be loaded for caching it to RAM memory,

```

Comment:
This Asset

Reply:
doesn't need to be converted, there are some functions in the engine which take ICPUBuffer as a nice way to represent a plain byte array.

Its used for storage of large arrays, so that storage can be decoupled from other objects such as meshbuffers, images, animations and shader source/bytecode.

Reply:
status?

Reply:
Solved